demo/app/components/drawer/drawer.component.html

```diff
@@ -23,6 +23,9 @@
       class="first-drawer"
       #drawer
     >
+      <ts-drawer-header>
+        <h4>HEADER</h4>
+      </ts-drawer-header>
       <div tsVerticalSpacing class="content">
```

Comment:
Maybe we make this class more specific? It seems like we're picking up some padding styles meant for primary pages:

```
router-outlet + * .content {
    padding: 2rem;
}
```